pkg/pillar/types/ifnametopci.go

```diff
@@ -79,6 +79,16 @@ func PCISameController(long1 string, long2 string) bool {
 	return ctrl1 == ctrl2
 }
 
+// PCIGetIOMMU returns IOMMU group tag as seen by the control domain
+func PCIGetIOMMU(long string) (string, error) {
```

Comment:
Putting "group" in the function name would have made me ask less questions.

Reply:
LOL -- good point ;-)

Reply:
Fixed